### DISCUSSION TITLE
Feature request: Be able to open the config file by clicking on the espanso icon on the down right side

### DISCUSSION THREAD

**Discussion by stephane-archer** [espanso/espanso#1225] [Feature requests and Ideas]
![image](https://user-images.githubusercontent.com/2981437/127275214-3ad0c0fe-5c25-4a9e-aa28-880cff100eb0.png)

Why not add, for convinience, a "Show config" button?

**Comment by federico-terzi** [comment]
Hey @stephane-archer,

Thank you for the feedback! Seems like a great addition, I'll add it to the roadmap :)

Cheers :)

**Comment by rochakagrawal** [comment]
I am wondering why the app doesn't has this and a GUI option to easily add / edit entries in the config table. Are there a lot of ppl who still prefer tinkering with configuration files?  I appreciate this is open source app etc. but with the level of marketing this app has on blogs / reddit - one would hope that a basic feature like GUI for adding / editing entries would be a must. Surprising even installing the app required manually moving executables to the bin file with no installer for Mac

**Comment by federico-terzi** [comment]
Hey @rochakagrawal,

Thank you for the feedback! A GUI is planned, and will probably be the next major milestone in the project (probably happening in 2022). This year the focus was to rewrite the core to solve many longstanding bugs and build a solid foundation for the years to come.

The reason why a GUI isn't available yet is that I have finite resources and time (I work on this in my spare time). If you'd like to see a GUI soon, you might consider donating or spreading the word about the project, as that really helps! :)

PS: the upcoming v2 will be much easier to install on macOS, as it's now a regular app bundle.

Cheers :)

**Comment by stephane-archer** [comment]
Hi @federico-terzi, do you have any news about this feature? I'm not sure where to find the roadmap :)

**Comment by federico-terzi** [comment]
@stephane-archer No update yet :) Should happen later this year
